docs/customize/vocabularies/users.md

```diff
@@ -3,17 +3,17 @@
 Users are not a vocabulary *per se*, but they are loaded in the same fashion,
 through the application data folder.
 
-This file contains a list of users to create, and is stored in the root of the
-`app_data` folder.
+The file `users.yaml` contains a list of users to create, and is stored in the
+root of the `app_data` folder.
 
 ```
 app_data/
 └── users.yaml
 ```
 
-If the file is provided but it is empty, no default user is created. If the
-file is not provided, InvenioRDM creates an admin user with email
-`admin@inveniosoftware.org` (and a random password).
+If the file is not provided, InvenioRDM creates an admin user with the email
+`admin@inveniosoftware.org` (and a random password). If the file is provided
+but it is empty, no default user is created.
```

Comment:
Let's actually move this after the "content of the file section", so that these edge features are highlighted after the normal features.

Removing "it" to sound more natural: "If the file is provided but is empty, no default user is created.